rel/overlay/etc/default.ini

```diff
@@ -249,6 +249,10 @@ iterations = 10 ; iterations for password hashing
 ; Settings for view indexing
 [couch_views]
 ; max_workers = 100
+; The maximum allowed key size emitted from a view for a document (in bytes)
+; key_size_limit = 8000
+; The maximum allowed value size emitted from a view for a document (in bytes)
+; key_size_limit = 64000
```

Comment:
`value_size_limit` I think?

The default are maximum as well? We could mention what happens if they are exceeded. I would guess we clip the config value to that max value/